### ISSUE TITLE
Don't use "here" as a link word

### ISSUE THREAD

**Issue by paulschreiber** [Automattic/newspack-popups#454]
https://github.com/Automattic/newspack-popups/blob/master/includes/class-newspack-popups.php#L545

```
Newspack Campaigns requires a custom configuration file, which is missing.
Please create this file following instructions found here.
```

This is a bad UX and AX practice. See:
* https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/
* https://webaccess.berkeley.edu/ask-pecan/click-here
